### PR TITLE
B 21042 prime create 3rd address

### PR DESCRIPTION
**Pull request by loganwc**
## [PR 2](https://github.com/transcom/mymove/pull/13796)

### CODE REVIEW

**Comment by loganwc** [comment]
The difference in changes is in the prime_v3 generated swagger file. New generated changes are lines >=954

File: swagger/prime_v3.yaml [hunk 1]
```diff
@@ -939,6 +977,16 @@ definitions:
           dropped off.
         allOf:
           - $ref: '#/definitions/Address'
+      hasTertiaryDestinationAddress:
```
Comment (pambecker) [inline swagger/prime_v3.yaml:980]: It appears this swagger file has 30+ extra row changes than your INT PRs.  did something need to go to main before this PR?

Comment (pambecker) [reply swagger/prime_v3.yaml:980]: There appears to have been a deletion of these in main somehow, these were added originally with 21054.  safe to add back.  Tested and was able to create an HHG and PPM shipment with 2 add'l addresses with success.

**Comment by pambecker** [comment]
> The difference in changes is in the prime_v3 generated swagger file. New generated changes are lines >=954

Found the PR that deleted them.  
https://github.com/transcom/mymove/pull/13718#pullrequestreview-2373745167

**Review by pambecker — APPROVED** [review]

**Pull request merged**